src/main/kotlin/no/nav/syfo/Bootstrap.kt

```diff
@@ -344,22 +366,28 @@ suspend fun handleMessage(
 
 fun finnTssIdFraInfotrygdRespons(sisteSmPeriode: TypeSMinfo.Periode?, behandler: Behandler): String? {
     if (sisteSmPeriode != null &&
-        behandler.etternavn.equals(sisteSmPeriode.legeNavn?.etternavn, true) &&
-        behandler.fornavn.equals(sisteSmPeriode.legeNavn?.fornavn, true)) {
+            behandler.etternavn.equals(sisteSmPeriode.legeNavn?.etternavn, true) &&
+            behandler.fornavn.equals(sisteSmPeriode.legeNavn?.fornavn, true)) {
         return sisteSmPeriode.legeInstNr?.toString()
     }
     return null
 }
 
+fun finnTssIdFraTSSRespons(tssSamhandlerInfoResponse: XMLTssSamhandlerData): String? {
+    return tssSamhandlerInfoResponse.tssOutputData.samhandlerODataB960?.enkeltSamhandler?.firstOrNull()?.samhandlerAvd125?.samhAvd?.find {
```

Comment:
Jeg skjønner at disse navnene antageligvis kommer fra et grensesnitt som er utenfor vår kontroll, men herregud, for noen navn 😅

Reply:
Jepp, eg måtte se hva som låg i eia, får lete fram dei riktige objektene, er tss modellen, vakkert.
XSDen, som forklare alt ligger her:
https://github.com/navikt/syfo-xml-codegen/blob/master/tssSamhandlerData/src/main/xsd/TSSskjema_31-05-10.xsd